src/KygekDev/IsOP/IsOP.php

```diff
@@ -25,18 +25,19 @@ class IsOP extends PluginBase {
     private const COMMAND_PERMISSION = "isop.command";
 
     public function onCommand(CommandSender $sender, Command $command, string $label, array $args) : bool {
-        if ($command->getName() !== "isop" || !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
+        if ($command->getName() !== "isop" or !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
```

Comment:
There is no big difference between `or` and ` || `
But `or` easier to read because it makes sense and `||` doesn't.

Reply:
Unfortunately this violates my coding style:
https://docs.kygek.team/coding-standards/php.html#example

This part:
```php
if ($output && $this->propertyTwo !== null) { // Prefer using strict comparison whenever possible
    echo implode(PHP_EOL, $this->propertyTwo);
}
```

Reply:
```suggestion
        if ($command->getName() !== "isop" || !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
```

Reply:
Sorry. I edited it.